portfolio/src/main/webapp/script.js

```diff
@@ -55,21 +55,49 @@ function pauseProjectChange() {
   clearTimeout(changeCurrProj);
 }
 
-/** Fetches comment(s) and updates the UI to display */
-function fetchCommentAndDisplay() {
+/** Fetches comment(s) and updates the UI to display them */
+function fetchCommentAndDisplay(numOfCommentsToDisplay) {
   fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
What would you think of a parameter to tell the server how many comments to return? Fetching the entire set could be wasteful and increase server or network latency needlessly if the client only wants a single comment.